src/SelectableBox/SelectableBoxSet.jsx

```diff
@@ -59,6 +64,7 @@ SelectableBoxSet.propTypes = {
   columns: PropTypes.number,
   /** A class that is be appended to the base element. */
   className: PropTypes.string,
+  mobileColumns: PropTypes.number,
```

Comment:
Let's add a comment above this prop to give it a description in the "Props API" table on the documentation website?

Reply:
Removed